target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -132,6 +132,11 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "2:wan" "3:lan" "4:lan"
 		;;
+	dlink,dap-1330-a1|\
+	dlink,dap-1365-a1)
+		ucidef_add_switch "switch0" \
+			"0@eth0" "1:lan:1"
```

Comment:
Why not just
ucidef_set_interface_lan "eth0"

Reply:
I guess the `Switch` menu item appearing on LuCI (for port status and VLAN configuration) is not a legitimate argument here, so it might as well be just LAN on all eth0.